packages/contracts/deploy/014-OVM_L1MultiMessageRelayer.deploy.ts

```diff
@@ -13,11 +14,20 @@ const deployFn: DeployFunction = async (hre) => {
     'Lib_AddressManager'
   )
 
-  await deployAndRegister({
+  const OVM_L1MultiMessageRelayer = await deployAndRegister({
     hre,
     name: 'OVM_L1MultiMessageRelayer',
     args: [Lib_AddressManager.address],
   })
+
+  // OVM_L2MessageRelayer *must* be set to multi message relayer address on mainnet.
+  if (hre.network.name.includes('mainnet')) {
```

Comment:
This seems fine to me as it will target one of many possible mainnet deployments at the same time

Reply:
Hmmm, thinking about it more, it is probably the case that we want to trigger this logic as much as we possibly can, because the downside of not triggering it when we do want to is much greated than the downside of triggering it when we do not.

Reply:
whoops, didn't see @tynes ' comment when I followed up, but sounds good 👍